src/components/AddPlaidBankAccount.js

```diff
@@ -93,7 +91,6 @@ class AddPlaidBankAccount extends React.Component {
 
         this.state = {
             selectedIndex: undefined,
-            password: '',
```

Comment:
It looks like we are still referencing this on line 116

```js
account, password: this.state.password, plaidLinkToken: this.props.plaidLinkToken,
```

I honestly can't remember, but is it safe to just remove this / is nothing expecting the password? 
What are we expecting on the API side here?

Reply:
Good spot, I thought I had removed this 😕

Yes, I believe it is safe to remove this. The password param is sent at each step, but it is only _required_ at step: `CompanyStep`. I'll remove the param and add some additional test steps to prove this is the case.

Reply:
Notice that we don't send password for the 'Connect manually' step either: https://github.com/Expensify/App/blob/993bc0bc52d310c66abfde122c47b7fa60a15122/src/pages/ReimbursementAccount/BankAccountStep.js#L79-L91

Reply:
I guess I am mostly curious here if this was an oversight or if the Web-Secure API was ever expecting this parameter? Probably we just went off of mockups here and assumed the password is required.

I think it's necessary when calling `BankAccount_Create` (for the PBA flow) but perhaps is not required for the VBA flow. I'm not too sure. I do want to point out that `SetupWithdrawalAccount` does accept a password parameter and validates it under certain conditions.

https://github.com/Expensify/Auth/blob/3ed205bafa8c39de56a77183732d3d6d0ce4429c/auth/command/SetupWithdrawalAccount.cpp#L130-L133

But unsure if that's something we need to worry about here.

Reply:
Oh yeah, that Auth line did make me question this too... But as my testing raised no issues I think it can be explained as an oversight or change.